editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -70,6 +70,7 @@ function Component(props: Props) {
     "medway",
     "newcastle",
     "southwark",
+    "st-albans",
```

Comment:
I knew we'd hit this GIS meme sooner or later 😅 

![image](https://github.com/theopensystemslab/planx-new/assets/20502206/6b50f174-4287-421c-a214-c3e9161c5af2)

This has made me realise that I used "stalbans" in infra code - I think if we switch it it might create a new CDN (and then they'll have to update CNAME records). We should aim to fix this when we renew their certs next year 👍 

This is correct though - our team slug is "st-albans". Might be another pointer towards using db flags for these types of values?

Reply:
Actually I think it's our team slug that is wrong and should be aligned to https://www.stalbans.gov.uk/ ! 

That has fewer consequences of updating immediately I think since service URLs aren't live yet? 

But, yes, db flag for toggling constraints is the way forward here - I'll add a column tomorrow!

Reply:
Yep I think the slug is fine to update as they aren't advertising the editor.planx.uk links anywhere - we might just need to cross check and update thinks like any links we have in Notion.